content/docs/anglescript/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Angelscript (Strata Hammer)
+title: Commands
 bookCollapseSection: true
 bookFlatSection: true
-weight: 100
+weight: 20
 ---
```

Comment:
Why has this been renamed?

Reply:
Sorry about this. I started out on my fork by deleting the Angelscript directory before I realized I needed to keep the _index.md file for the website. So I just copied from the commands _index.md (forgetting to change the title). I should've checked diff compared to this repo before making this pr.

Reply:
All good, thanks for this!